pkgs/tools/security/aeskeyfind/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchurl
+}:
+
+stdenv.mkDerivation rec {
+  pname = "aeskeyfind";
+  version = "1.0";
+
+  src = fetchurl {
+    url = "https://citpsite.s3.amazonaws.com/memory-content/src/aeskeyfind-${version}.tar.gz";
+    sha256 = "sha256-FBflwbYehruVJ9sfW+4ZlaDuqCR12zy8iA4Ev3Bgg+Q=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    cp aeskeyfind $out/bin
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Locates 128-bit and 256-bit AES keys in a captured memory image";
+    homepage = "https://citp.princeton.edu/our-work/memory/";
+    maintainers = with maintainers; [ fedx-sudo ];
+  };
```

Comment:
What about `meta.license`?

Reply:
There is no license that I can find.

Reply:
That's a problem.

Reply:
```suggestion
  meta = with lib; {
    description = "Locates 128-bit and 256-bit AES keys in a captured memory image";
    homepage = "https://citp.princeton.edu/our-work/memory/";
    license = unfree;
    maintainers = with maintainers; [ fedx-sudo ];
  };
```

Reply:
The source tarball contains a `LICENSE` file.

Reply:
Huh, I literally opened the tarball but didn't see the license file. Thanks.

Reply:
```suggestion
  meta = with lib; {
    description = "Locates 128-bit and 256-bit AES keys in a captured memory image";
    homepage = "https://citp.princeton.edu/our-work/memory/";
    license = bsd3;
    maintainers = with maintainers; [ fedx-sudo ];
  };
```